examples/vmi-alpine-efi.yaml

```diff
@@ -12,10 +12,6 @@ spec:
       - disk:
           bus: virtio
         name: containerdisk
-    firmware:
-      bootloader:
-        efi:
-          secureBoot: false
```

Comment:
I don't follow why was this file changed, could you please elaborate?

Reply:
Its not related, sorry for the noise.
Fixed.